go/firewallsso/checkpoint.go

```diff
@@ -32,13 +35,13 @@ func (fw *Checkpoint) Start(ctx context.Context, info map[string]string, timeout
 // Build the RADIUS packet for an SSO start
 func (fw *Checkpoint) startRadiusPacket(ctx context.Context, info map[string]string, timeout int) *radius.Packet {
 	r := radius.New(radius.CodeAccountingRequest, []byte(fw.Password))
-	r.Set("Acct-Status-Type", uint32(1))
-	r.Set("Acct-Session-Id", "acct_pf-"+info["mac"])
-	r.Set("User-Name", info["username"])
-	r.Set("Session-Timeout", uint32(timeout))
-	r.Set("Called-Station-Id", "00:11:22:33:44:55")
-	r.Set("Framed-IP-Address", net.ParseIP(info["ip"]))
-	r.Set("Calling-Station-Id", info["mac"])
+	rfc2866.AcctStatusType_Add(r, rfc2866.AcctStatusType(1))
```

Comment:
use constant instead for readability: `AcctStatusType_Value_Start`

Reply:
use rfc2866.AcctStatusType_Value_Start instead of rfc2866.AcctStatusType(1)